Automattic/wp-calypso
# ci(teamcity): Move builds to Webapp subproject

### Background

Part of #51132

### Changes proposed in this Pull Request

* Used TeamCity UI to get the `uuid` for each build and added it.
* Move build `Run unit tests` to `Web app` subproject
* Move build `Check code style for branches` to `Web app` subproject
* Move build `Build docker image` to `Web app` subproject

Moving the config has been done with IntelliJ so it adjusts the imports automatically.

### Testing instructions

Check any TeamCity build for this branch and verify it doesn't comply about errors in the DSL.
